mowgli_etl/pipeline/wdc/wdc_generic_size.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass
+from dataclasses_json import dataclass_json
+from typing import Optional
+from mowgli_etl.pipeline.wdc.wdc_product_dimensions import WdcProductDimensions
+
+
+@dataclass_json
+@dataclass
+class WdcGenericSize:
+    name: str
```

Comment:
Not entirely clear what these fields are. I'll look further down, but wouldn't hurt to add some documentation of the fields.